README.rst

```diff
@@ -1,131 +1,156 @@
+music, art, gameplay mechanics, and codebase. While some familiarity with Tōhō
+is helpful, the gameplay can be enjoyed on its own without prior knowledge of
+the series.
+
+For more information on dōjin culture,
+`click here <https://en.wikipedia.org/wiki/D%C5%8Djin>`__.
 
 Installation
 ------------
 
-Dependencies
-^^^^^^^^^^^^
+You can find precompiled binaries of the complete game on the
+`Releases <https://github.com/taisei-project/taisei/releases>`__ page on
+GitHub, available for Windows (x86/x64), Linux, and macOS.
+
+An experimental build for Nintendo Switch (homebrew) also exists (use at your
+own risk).
+
+You can also play our experimental WebGL build through your web browser
+`here <https://play.taisei-project.org/>`__. (Chromium-based browsers and
+Firefox supported.)
 
--  OpenGL >= 3.3 or OpenGL ES >= 3.0 or OpenGL ES >= 2.0 (with some extensions)
--  `cglm <https://github.com/recp/cglm>`__ >= 0.7.8
+Source Code & Development
+-------------------------
+
+Basic Dependencies
+^^^^^^^^^^^^^^^^^^
+
+Run-Time Dependencies
+_____________________
+
+Required
+********
+
+-  OpenGL >= 3.3, or OpenGL ES >= 3.0
 -  SDL2 >= 2.0.10
--  freetype2
 -  libpng >= 1.5.0
 -  libwebpdecoder >= 0.5 or libwebp >= 0.5
 -  libzip >= 1.5.0 (>= 1.7.0 recommended)
 -  libzstd >= 1.4.0
+-  freetype2
 -  opusfile
 -  zlib
 
-Optional:
+Optional
+********
 
 -  OpenSSL (for a better SHA-256 implementation; used in shader cache)
 -  SPIRV-Cross >= 2019-03-22 (for OpenGL ES backends)
 -  libshaderc (for OpenGL ES backends)
+-  `ANGLE <https://github.com/google/angle>`__ (for ANGLE support, useful for Windows builds)
 -  GameMode headers (Linux only; for automatic `GameMode
    <https://github.com/FeralInteractive/gamemode>`__ integration)
 
-Build-only dependencies
-^^^^^^^^^^^^^^^^^^^^^^^
+Build-Time Dependenices
+_______________________
 
 -  Python >= 3.6
--  meson >= 0.53.0
--  [python-zstandard](https://github.com/indygreg/python-zstandard)
+-  `python-zstandard <https://github.com/indygreg/python-zstandard>`__ >= 0.11.1
+-  meson >= 0.56.0
+-  ``gcc`` or ``clang``
 
-Optional:
-
--  docutils (for documentation)
-
-Obtaining the source code
-^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Stable releases
-"""""""""""""""
-
-You can find the source tarballs at the
-`Releases <https://github.com/taisei-project/taisei/releases>`__ section on
-Github. **Do not** grab Github's auto-generated source archives, those do not
-contain the required submodules. This only applies for versions v1.3 and above.
+Obtaining Source Code
+^^^^^^^^^^^^^^^^^^^^^
 
-Latest code from git
-""""""""""""""""""""
+You can obtain the source code of the stable releases from the
+`Releases <https://github.com/taisei-project/taisei/releases>`__ page on
+GitHub, alongside the binaries. (Note that you **MUST** use the releases page,
+as the ``Download ZIP`` link on the main repo *does not* include all the code
+necessary to build the game.)
```

Comment:
I think it's good to say that it doesn't include submodules, specifically, to provide some context as to why this is the case to the more git savvy reader. I would say something like "as Github does not automatically include the necessary submodules in those packages", to imply that this mess isn't due to our choice.